src/Clean-Energy-PA-Site/UserRegistration/tests/test_activate_view.py

```diff
@@ -45,9 +49,10 @@ def test_activate_invalid_token(self):
 
     def test_activate_exception_path(self):
         """Test ID 48: Test covers the exception path of activate view"""
-        url = reverse(
-            "UserRegistration:activate",
-            kwargs={"uidb64": "invalid", "token": "invalid"},
-        )
-        self.client.get(url)
+        response = self.client.post(self.activate_url)
         self.assertRaises(Exception)
+
+    def test_activate_exception_path(self):
+        """Test ID TBD: Test covers the non-POST path"""
```

Comment:
Small comment, need to resolve TBD before the merge.

Reply:
Applied next available test ID
```suggestion
        """Test ID 85: Test covers the non-POST path"""
```